web-bundle/src/main/java/com/graphhopper/resources/RouteResource.java

```diff
@@ -190,9 +188,7 @@ public Response doPost(@NotNull GHRequest request, @Context HttpServletRequest h
                 + ", custom_model: " + request.getCustomModel();
 
         if (ghResponse.hasErrors()) {
-            MultiException ex = new MultiException(ghResponse.getErrors());
-            logger.error(logStr, ex);
```

Comment:
Strangely we logged this as an error for POST , but as info for GET. Either way we already log the error in MultiExceptionMapper so I removed this log.